examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+"""
```

Comment:
sphinx build make clean should clean up after any sphinx-gallery examples files and folders:

- multiple files are placed directly into the `examples` folder (this are ignored by git)
- application data folders including `.dipy` and `AFQ_data` (these are not part of source)

Depending on what files are present this may impact local builds

Reply:
Yes. See #485